packages/cardano-services/src/Utxo/DbSyncUtxoProvider/mappers.ts

```diff
@@ -0,0 +1,52 @@
+import { Cardano } from '@cardano-sdk/core';
+import { UtxoModel } from './types';
+
+const generateAssetId = (policy: string, name: string) =>
+  Cardano.AssetId(policy + (!name || name === '' ? '\\x' : name));
+
+/**
+ * Indexes the given utxos by its coin id in order to generate core utxo objects
+ *
+ * @param {UtxoModel[]} utxos  obtained utxos rows
+ * @returns {Cardano.Utxo[]} an array of core utxo objects
+ */
+export const toUtxoByAddressesResponse = (utxos: UtxoModel[]): Cardano.Utxo[] => {
+  const utxosByCoin = utxos.reduce((coins, current) => {
+    const coinId = `${current.tx_id}:${current.index}`;
```

Comment:
```suggestion
    const utxoId = `${current.tx_id}:${current.index}`;
```
Might be worth converting to a function and hoisting to a util